test/sharness/t0130-multinode.sh

```diff
@@ -53,6 +53,18 @@ check_file_fetch() {
 	'
 }
 
+test_resolve() {
+	_nid="$1"; shift
```

Comment:
This pattern is useful whenever you want to have a variable number of parameters in the end. You can take each mandatory parameter with `foo="$1"; shift` and you have all the remaining parameters (zero or more) in `"$@"` afterwards. I used that in a couple of the other tests, so I used the same style for consistency despite not using `"$@"` in this specific function.

In scripts that use `set -e`, the `shift` also acts as an additional sanity check that the mandatory parameter is actually there. On the other hand, with `set -u` just the `"$1"` expansion covers that. But this is irrelevant here as sharness suites do not use either.